doc/source/quickstart/index.rst

```diff
@@ -14,14 +14,15 @@ worked examples.
 
 To execute the quickstart interactively, you have a couple of options: 1) run
 the notebook from your own system or 2) run it from the url
-https://demo.use.yt. Option 1 requires an existing installation of yt (see
+https://girder.hub.yt/#raft/5b5b4686323d12000122aa8a.
```

Comment:
demo.use.yt should redirect to girder.hub.yt. However, I like that you've linked to the specific raft that has all of the quickstart notebooks in it. That way new users won't wonder which raft to click on.